pinecone/run_integration_test_suites.go

```diff
@@ -0,0 +1,52 @@
+package pinecone
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func RunSuites(t *testing.T) {
```

Comment:
I wonder if there's a way to add an additional parameter here like a `pod` vs. `serverless` enum. We could then directly control what's run when each `client_test.go` or `index_connection_test.go` runs `RunSuites()`, so you could control behavior more directly via integration test file. That feels like it might be nice.

Like you said, there's a lot of ways we could take this and we should probably just take this first step and then play around.